lib/livebook/hubs/team_client.ex

```diff
@@ -66,20 +66,32 @@ defmodule Livebook.Hubs.TeamClient do
   ## GenServer callbacks
 
   @impl true
-  def init(%Team{} = team) do
+  def init(%Team{offline: nil} = team) do
+    derived_keys = Teams.derive_keys(team.teams_key)
+
     headers = [
-      {"x-user", to_string(team.user_id)},
-      {"x-org", to_string(team.org_id)},
-      {"x-org-key", to_string(team.org_key_id)},
-      {"x-session-token", team.session_token}
+      {"x-user", to_string(team.user_id || 0)},
+      {"x-org", to_string(team.org_id || 0)},
+      {"x-org-key", to_string(team.org_key_id || 0)},
+      {"x-session-token", to_string(team.session_token)}
     ]
 
-    derived_keys = Teams.derive_keys(team.teams_key)
-
     {:ok, _pid} = Connection.start_link(self(), headers)
     {:ok, %__MODULE__{hub: team, derived_keys: derived_keys}}
   end
 
+  def init(%Team{} = team) do
+    derived_keys = Teams.derive_keys(team.teams_key)
+
+    {:ok,
+     %__MODULE__{
+       hub: team,
+       connected?: true,
```

Comment:
This is not really true, right? Maybe we should have a `offline?` attribute instead and check for it where necessary?

Reply:
In this case, I think we should show a red indicator or no indicator at all. :)